packages/voice/src/rest/v1/voice-domain-api.ts

```diff
@@ -87,15 +87,15 @@ export class VoiceDomainApi implements Api {
       };
       this.client = new ApiFetchClient(apiClientOptions);
       const region: VoiceRegion = this.sinchClientParameters.voiceRegion || VoiceRegion.DEFAULT;
-      this.client.apiClientOptions.basePath = this.buildBasePath(region);
+      this.client.apiClientOptions.hostname = this.sinchClientParameters.voiceHostname ?? this.buildBasePath(region);
```

Comment:
Same as for Conversation and Sms: Could require a note for SDK's user (_region is not used when hostname is_)
And same question as for Conversation:  is it possible to override the 2 supported endpoints: the general  one for `Conversation` and the specific one for `Applications` endpoints ?

Reply:
Hehe, I've replied just above already 😁

Reply:
Fixed 🙂